modules/tinymce/src/themes/silver/test/ts/browser/editor/bespoke/NamedStyleSelectFormatsTest.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  ApproxStructure, Assertions, Chain, Logger, Pipeline, Step, UiFinder
+} from '@ephox/agar';
+import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { TinyLoader } from '@ephox/mcagar';
+import { SugarBody } from '@ephox/sugar';
+import Theme from 'tinymce/themes/silver/Theme';
+import * as MenuUtils from '../../../module/MenuUtils';
+
+interface StyleSelectMenuItem {
+  element: string;
+  title: string;
+}
+
+UnitTest.asynctest('browser.tinymce.themes.silver.bespoke.NamedStyleSelectFormatTest', (success, failure) => {
+  Theme();
+  TinyLoader.setup(
```

Comment:
This formatting is a bit weird? Normally this would be on one line. eg:
```
TinyLoader.setup((editor, onSuccess, onFailure) => {
```

Reply:
That became even stranger a lot of other tests in silver theme has that same indentation. SilverBespokeButtonsTest.ts, ContextFormTest.ts, DisableTest.ts etc etc. If we want to change that indention then all of them should change to keep it consistent.

Reply:
Right, that is an issue. Ignore for now and we'll fix that another time :+1: